src/simulation/LoadGenerator.cpp

```diff
@@ -1250,30 +1343,26 @@ LoadGenerator::invokeSorobanLoadTransaction(uint32_t ledgerNum,
     ihf.invokeContract().args = {guestCyclesU64, hostCyclesU64, numEntriesU32,
                                  kiloBytesPerEntryU32};
 
-    // We don't have a good way of knowing how many bytes we will need to read
-    // since the previous invocation writes a random number of bytes, so use
-    // upper bound
-    const uint32_t leOverhead = 75;
-    auto readBytesUpperBound =
-        (cfg.kiloBytesPerDataEntryHigh * 1024 + leOverhead) * numEntries +
-        mContactOverheadBytes;
-    auto writeBytes = (kiloBytesPerEntry * 1024 + leOverhead) * numEntries;
-
     // baseInstructionCount is a very rough estimate and may be a significant
     // underestimation based on the IO load used, so use max instructions
     resources.instructions = networkCfg.mTxMaxInstructions;
-    resources.readBytes = readBytesUpperBound;
-    resources.writeBytes = writeBytes;
+
+    // We don't have a good way of knowing how many bytes we will need to read
+    // since the previous invocation writes a random number of bytes, so use
+    // upper bound
+    resources.readBytes = invokeCfg.ioKiloBytesHigh * 1024;
```

Comment:
Why did you change this back to the max value?

Reply:
This is the same value that we had before. Previously this was 
```
readBytes = (cfg.kiloBytesPerDataEntryHigh * 1024 + leOverhead) * numEntries + mContactOverheadBytes;
```
Since the config parameter now dictates total TX io size instead of the size of individual data entries, we can simplify it. Note that this is the LoadGen config max, not the network settings max. Unfortunately we have a lot of different config types floating around so it's a bit confusing.

Reply:
Ah, right, I've got confused by this a bit.